client/utils/tags.test.tsx

```diff
@@ -29,7 +29,7 @@ describe('Tags', () => {
   it('injectConditionalTags should inject cap and chinese gap for chinese homeLocation', () => {
     const injectedTags = injectConditionalTags(
       [],
-      'https://chinese.freecodecamp.org'
+      'https://freecodecamp.org/chinese'
     );
     expect(injectedTags.length === 2).toBeTruthy();
```

Comment:
This test will continue to fail, because the function uses the `env` locales which are set to ENGLISH in the CI.